lib/console.js

```diff
@@ -29,7 +29,7 @@ const {
     ERR_INVALID_ARG_VALUE,
   },
 } = require('internal/errors');
-const { previewMapIterator, previewSetIterator } = require('internal/v8');
+const { previewEntries } = process.binding('util');
```

Comment:
can we put them on an internalBinding instead?

Reply:
Any concrete preferences? The most stable part of `process.binding('util')` (type checking) is runtime-deprecated in Node 10, so we might be able to move the entire util binding to `internalBinding('util')` in Node 11 anyway…

Reply:
whatever gets it done is fine with me, i just want to see process.binding stop existing :)